examples/normalize.rs

```diff
@@ -1,5 +1,5 @@
-use kornia_rs::image::Image;
-use kornia_rs::io::functional as F;
+use kornia_image::Image;
+use kornia_io::functional as F;
```

Comment:
why not keep using the top lib in the examples?

Reply:
still figuring out how to recover the top level api -- maybe it's gonna be a big breaking change. Not clear yet

Reply:
Fixed